cmd/src/doc.go

```diff
@@ -58,7 +58,7 @@ Examples:
 			"config":       &configCommands,
 			"extensions":   &extensionsCommands,
 			"extsvc":       &extsvcCommands,
-			"lsif":         &lsifCommands,
+			"code-intel":   &codeintelCommands,
```

Comment:
Is there a way to keep supporting `src lsif` but hide it from `-help`? I'm not convinced this backwards breaking change is worth it because it punishes to customers who have gone through the effort of setting up precise intel in their CI pipelines. We only introduced `src code-intel` a few months ago, I'd expect the deprecation cycle to last at least 1-2 years for an enterprise product like our and I honestly think it's fine to support `src lsif` forever as long as it doesn't cause a maintenance burden.

Reply:
How is this backwards breaking? It changes the docs on dcos.sourcgraph.com from what you see on the left to what you see on the right. It doesn't change anything about how the commands or src-cli works.

![screenshot_2022-08-22_10 40 53@2x](https://user-images.githubusercontent.com/1185253/185878401-b199c852-e22a-4d32-8aa4-b5297369822e.png)

Reply:
Yeah, there's no need to remove the old one, there is no major benefit of making the breaking change.